### ISSUE TITLE
Is it necessary to login to the developer portal on every build?

### ISSUE THREAD

**Issue by andrewtimberlake** [HipByte/motion-provisioning#1]

**Comment by MarkVillacampa** [comment]
If you have to login on every build, then it is a bug.

Can you paste:
1- The full output when you run "rake"
2- The contents of your "provisioning" folder (e.g. executing "ls provisioning")

**Comment by amirrajan** [comment]
Can you elaborate on this?

**Comment by andrewtimberlake** [comment]
Rake output
```
      Info Logging into the Developer Portal with email '<email address>'.
      Info Selected team 'XXXXXXXXXXXXXX. (YYYYYYYYY)'.
      Info Using certificate 'Andrew Timberlake'.
      Info Using provisioning profile '(MotionProvisioning) com.myapp.identifer ios development'.
     Build ./build/iPhoneSimulator-7.1-Development
   Compile ./app/app_delegate.rb
      Link ./build/iPhoneSimulator-7.1-Development/MYAPP.app/MYAPP
    Create ./build/iPhoneSimulator-7.1-Development/MYAPP.app/Info.plist
    Create ./build/iPhoneSimulator-7.1-Development/MYAPP.app.dSYM
      Copy ./build/iPhoneSimulator-7.1-Development/MYAPP.app.dSYM
  Simulate /Users/andrew/dev/myapp/ios/build/iPhoneSimulator-7.1-Development/MYAPP.app
```

provisioning folder contents
```
config.yml
ios_developer_certificate.cer
com.myapp.identifier_ios_development_provisioning_profile.mobileprovision
```

**Comment by MarkVillacampa** [comment]
The name of the certificate file is wrong, it should be `ios_development_certificate.cer` not `ios_developer_certificate.cer`.

Can you show me the part of the Rakefile where you configure motion-provisioning?

**Comment by andrewtimberlake** [comment]
```
  app.development do
    app.info_plist['NSAppTransportSecurity'] = {"NSAllowsArbitraryLoads" => true}

    app.codesign_certificate = MotionProvisioning.certificate(
      type: :development,
      platform: :ios
    )
    app.provisioning_profile = MotionProvisioning.profile(
      bundle_identifier: app.identifier,
      app_name: app.name,
      platform: :ios,
      type: :development
    )
  end
```

**Comment by MarkVillacampa** [comment]
Looks like we're checking the private key file is present here:

https://github.com/HipByte/motion-provisioning/blob/master/lib/motion-provisioning/certificate.rb#L34

We well tweak that behavior. Meanwhile you can make it not log in every time by running `touch provisioning/ios_development_private_key.p12`. If that doesnt work, try "developer" instead of "developent".

**Comment by andrewtimberlake** [comment]
Thanks, I'll give that a try tomorrow.
One thing, the files in there were generated from the first run so motion-provision named them?

Andrew

**Comment by MarkVillacampa** [comment]
Yes, motion-provisioning named them. I'll try to figure out where the wrong name comes from too

**Comment by andrewtimberlake** [comment]
Whether I create a file `provisioning/ios_development_private_key.p12` then all works.

**Comment by MarkVillacampa** [comment]
Should be fixed with e053af3ac81b54037bc6abcdd4c792d9c872bda2

Please update to motion-provisioning 0.0.3

**Comment by andrewtimberlake** [comment]
Thank you. It is working perfectly.
